pkgs/applications/version-management/bazaar/default.nix

```diff
@@ -21,11 +20,7 @@ python2Packages.buildPythonApplication rec {
   patches = [
     # Bazaar can't find the certificates alone
     ./add_certificates.patch
-    (fetchpatch {
-      url = "https://bazaar.launchpad.net/~brz/brz/trunk/revision/6754";
```

Comment:
It looks like the .patch file is now at https://bazaar.launchpad.net/~brz/brz/trunk/diff/6754?context=3 instead. Can we just fetchpatch that URL instead of embedding the patch in the repo?

Reply:
Ah, it looks like @Izorkin did that in master. I guess we can just carry it over then, but in general I'd treat embedding literal patches in the repo as a last resort

Reply:
Another thought: if this could stay a FO derivation as `fetchpatch`, it wouldn't cause a giant rebuild 😄